### PR TITLE
Feature/all issues

### PR DESCRIPTION
**Pull request by datka-mcoder**

### CODE REVIEW

File: src/main/java/kg/peaksoft/taskTrackerb6/db/model/Estimation.java [hunk 1]
```diff
@@ -37,8 +37,8 @@ public class Estimation {
     @Column(length = 10000)
     private String text;
 
-    @OneToOne(cascade = {DETACH, REFRESH, PERSIST, MERGE})
-    private Card card;
+//    @OneToOne(cascade = {DETACH, REFRESH, PERSIST, MERGE})
+//    private Card card;
```
Comment (datka-mcoder) [inline src/main/java/kg/peaksoft/taskTrackerb6/db/model/Estimation.java:41]: remove

File: src/main/java/kg/peaksoft/taskTrackerb6/api/AllIssuesApi.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import kg.peaksoft.taskTrackerb6.enums.LabelsColor;
+import lombok.RequiredArgsConstructor;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@RestController
+@RequestMapping("api/allIssues")
```
Comment (i-Coderr) [inline src/main/java/kg/peaksoft/taskTrackerb6/api/AllIssuesApi.java:17]: Rename to api/issues

File: src/main/java/kg/peaksoft/taskTrackerb6/api/AllIssuesApi.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+@RequiredArgsConstructor
+@CrossOrigin(origins = "*", maxAge = 3600)
+@Tag(name = "All issues API", description = "All endpoints of all issues")
```
Comment (i-Coderr) [inline src/main/java/kg/peaksoft/taskTrackerb6/api/AllIssuesApi.java:20]: Issues Api

File: src/main/java/kg/peaksoft/taskTrackerb6/api/AllIssuesApi.java [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+public class AllIssuesApi {
+
+    private final AllIssuesService service;
+
+    @Operation(summary = "Get all workspace cards", description = "Get all workspace cards by workspace id")
+    @GetMapping("all-issues/{workspaceId}")
```
Comment (i-Coderr) [inline src/main/java/kg/peaksoft/taskTrackerb6/api/AllIssuesApi.java:26]: cards

File: src/main/java/kg/peaksoft/taskTrackerb6/api/AllIssuesApi.java [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    public List<AllIssuesResponse> allIssues(@PathVariable Long workspaceId) {
+        return service.allIssues(workspaceId);
+    }
+
+    @Operation(summary = "Filter by created date", description = "Filter cards by created date")
+    @GetMapping("dates/{id}")
```
Comment (i-Coderr) [inline src/main/java/kg/peaksoft/taskTrackerb6/api/AllIssuesApi.java:32]: filter-dates/{id}

File: src/main/java/kg/peaksoft/taskTrackerb6/api/AllIssuesApi.java [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    public SearchCard filterByCreatedDate(@PathVariable Long id,
+                                          @RequestParam("fromDate") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate fromDate,
+                                          @RequestParam("to") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate to) {
+        return service.filterByCreatedDate(id, fromDate, to);
+    }
+
+    @Operation(summary = "Filter by label's color", description = "Filter cards by label's color")
+    @GetMapping("colors/{id}")
```
Comment (i-Coderr) [inline src/main/java/kg/peaksoft/taskTrackerb6/api/AllIssuesApi.java:40]: filter-colors/{id}

File: src/main/java/kg/peaksoft/taskTrackerb6/dto/request/CardDatesRequest.java [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+import java.time.LocalDate;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class CardDatesRequest {
+
+    private Long workspaceId;
+    private LocalDate fromDate;
```
Comment (i-Coderr) [inline src/main/java/kg/peaksoft/taskTrackerb6/dto/request/CardDatesRequest.java:17]: from

File: src/main/java/kg/peaksoft/taskTrackerb6/db/repository/UserRepository.java [hunk 8]
```diff
@@ -31,5 +31,9 @@ public interface UserRepository extends JpaRepository<User, Long> {
             + " or u.email like :text "
     )
     List<User> globalSearch(@Param("text") String text);
+
+    @Query("select new kg.peaksoft.taskTrackerb6.dto.response.CardMemberResponse(u.id, u.photoLink)" +
```
Comment (i-Coderr) [inline src/main/java/kg/peaksoft/taskTrackerb6/db/repository/UserRepository.java:35]: u.photoLink please change photoLink to image in User entity class

File: src/main/java/kg/peaksoft/taskTrackerb6/dto/response/CardMemberResponse.java [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+@Setter
+@AllArgsConstructor
+public class CardMemberResponse {
+
+    private Long id;
+    private String image;
+
+    public CardMemberResponse(User user) {
+        this.id = user.getId();
+        this.image = user.getPhotoLink();
```
Comment (i-Coderr) [inline src/main/java/kg/peaksoft/taskTrackerb6/dto/response/CardMemberResponse.java:18]: this.image = user.getImage();

**Review by i-Coderr — COMMENTED** [review]
Commented at 4.04 pm

**Review by i-Coderr — COMMENTED** [review]
Well done

**Pull request merged**